renovate.json

```diff
@@ -1,37 +1,18 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", "schedule:daily", ":disableRateLimiting"],
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":disableRateLimiting"
+  ],
   "schedule": "before 5am every weekday",
   "timezone": "Europe/Berlin",
-  "packageRules": [
-    {
-      "matchUpdateTypes": ["minor", "patch"],
-      "matchCurrentVersion": ">= 1.0.0",
-      "automerge": true
-    },
-    {
-      "matchDepNames": ["node", "pnpm"],
-      "enabled": false
-    },
-    {
-      "matchPackagePrefixes": ["@types/"],
-      "automerge": true,
-      "major": {
-        "automerge": false
-      }
-    }
-  ],
-  "lockFileMaintenance": {
-    "enabled": true,
-    "automerge": false
-  },
-  "major": {
-    "automerge": false
+  "rebaseWhen": "never",
+  "vulnerabilityAlerts": {
+    "labels": [
+      "npm dependencies, renovate, security"
+    ],
+    "automerge": true
   },
```

Comment:
```suggestion
  },
  "osvVulnerabilityAlerts": true,
```
I would also like to include this for additional vulnerability checks. (https://docs.renovatebot.com/configuration-options/#osvvulnerabilityalerts)

Reply:
good point. thx.